modules/fluid_properties/include/userobjects/SalineFluidProperties.h

```diff
@@ -0,0 +1,139 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "SinglePhaseFluidProperties.h"
+
+#ifdef SALINE_ENABLED
+#include "default_data_store.hh"
+#include "thermophysical_properties.hh"
+#endif
+
+class SalineFluidProperties : public SinglePhaseFluidProperties
+{
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Woverloaded-virtual"
```

Comment:
that makes me a bit uneasy

Reply:
yeah this is not new, it s something we will fix everywhere when we get to fixing it